packages/idx/src/property/admin/search.filter.component.html

```diff
@@ -9,31 +9,37 @@ <h3>Location</h3>
             <div class="location-specs" data-layout="row">
                 <div class="search-input" data-flex="50" data-flex-gt-sm="25">
                     <md-input-container class="md-block minimal">
-                        <label>City, Neighborhood, Zip</label>
-                        <input data-ng-model="options.query.Location"
+                        <label>City</label>
+                        <input data-ng-model="options.query.City"
```

Comment:
Did we change `Location` back to `City`?

Reply:
we also need to add a section for Location since the frontend uses Location now.